lib/referral.ts

```diff
@@ -0,0 +1,60 @@
+import { AxiosRequestConfig } from "axios";
+
+export enum ReferralStatus {
+  PENDING = "pending",
+  REJECTED = "rejected",
+  ACCEPTED = "accepted",
+  COMPLETE = "complete",
+  CANCELLED = "cancelled"
+}
```

Comment:
The only status that I am aware of that is supported is `pending` (and based on the ReferralFixture in the Partner API, it is the only one it supports). The rest of the status have not yet been defined by the business - they are currently working on the changes to Connect to define these.

Reply:
At the moment, we don't know what the other status will be, so we'll need to only support `pending` or we put this PR on hold until Connect complete their analysis (a couple of weeks I would expect). 🤔

Reply:
Alright, I'll leave this on hold until Connect decide on the statuses